### PR TITLE
Fixed Sails 0.11 / socket.io 1.x compatibility

### PR DESCRIPTION
**Pull request by psi-4ward**

### CODE REVIEW

File: src/service/angular-sails.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ angular.module('ngSails').provider('$sails', function () {
     this.responseHandler = undefined;
 
     this.$get = ['$q', '$timeout', function ($q, $timeout) {
-        var socket = io.connect(provider.url),
+        var socket = io.sails.connect(provider.url),
```
Comment (TheSharpieOne) [inline src/service/angular-sails.js]: Can you make this backward compatible with something like `(io.sails || io).connect(provider.url)`?

**Comment by TheSharpieOne** [comment]
LGTM

**Comment by TheSharpieOne** [comment]
@psi-4ward does this fix relive the issue you had here: https://github.com/janpantel/angular-sails/issues/34#issuecomment-66605243 ?

**Comment by psi-4ward** [comment]
No i dont think so

**Pull request merged**